docs/technical-documentation/rest-create.md

```diff
@@ -0,0 +1,195 @@
+## Creating resources, media and files - POST/PUT Requests 
+
+## Authorization
+
+You will need to use one of the configured authorization methods to create content, media and/or files.
+
+These are defined under [Authorization](./using-rest-endpoints.md#authorization) on the overview.
+
+## Content (Nodes)
+
+![REST configuration](../assets/rest-node-configuration.png)
+
+The above setup shows that you can perform a POST request against a node at the `/node` endpoint with a body in the JSON format.
+
+To create a node you need to provide two elements in your message body. The node type and any _required_ field values.
+
+For the islandora_demo included Repository Item these are:
+
+1. A type - this tells Drupal what content type we are creating
+1. A title - this is a required field of all nodes.
+1. A model - this is a required by Islandora to tell the type of object (ie. Image, Audio, Collection)
+
+A good way to make your first POST request is to perform a GET request against an existing node and erase all the extra content.
+
+You can find more information about [GET requests here](./rest-get.md)
+
+Again we are using the json format. 
+
+&#x1F34E; For example `curl -X GET 'http://localhost:8000/node/3?_format=json`
+
+Look for the **type** element
+
+```
+"type" : [
+  {
+    "target_id" : "islandora_object",
+    "target_type" : "node_type",
+    "target_uuid" : "62189bec-3ef3-4196-b847-b17e5ce61fd5"
+  }
+]
+```
+
+In our example "islandora_object" is the type of the default "Repository Item", if you have created a new type you will have a different target_id.
+
+You will not need the `target_uuid`.
+
+Next look for the **title** element
+
+```
+"title" : [
+  {
+    "value" : "An example Islandora object"
+  }
+]
+```
+
+Lastly look for the **field_model** element
+
+```
+"field_model": [
+  {
+    "target_id": 24,
+    "target_type": "taxonomy_term",
+    "target_uuid": "e7560b68-e95a-4e76-9671-2a3041cd9800",
+    "url": "\\/taxonomy\\/term\\/24"
+  }
+]
+```
+
+You can find the models by browsing the taxonomy terms available at `http://localhost:8000/admin/structure/taxonomy/manage/islandora_models/overview`
+
+24 is an "Image", but let's create a collection which is 23.
+
+So the body of the request will be:
+```
+{
+  "type": [
+    {
+      "target_id": "islandora_object",
+      "target_type": "node_type"
+    }
+  ],
+  "title": [
+    {
+      "value": "Created a collection with POST"
+    }
+  ],
+  "field_model": [
+    {
+      "target_id": 23,
+      "target_type": "taxonomy_term"
+    }
+  ]
+}
+```
+
+**Note**: You **must** include a **Content-type** header of type `application/json` on your request
```

Comment:
application/json is probably what we would recommend.  However, xml is probably supported as well.

Reply:
maybe

> **must** include an appropriate **Content-type** header for the format you're requesting